buildflow/core/app/runtime/autoscaler.py

```diff
@@ -25,49 +11,96 @@
 from buildflow.core.options.runtime_options import AutoscalerOptions
 from buildflow.core.processor.processor import ProcessorType
 
-# TODO: Make this configurable
-_TARGET_UTILIZATION = 0.5
+_BACKLOG_BURN_THRESHOLD = 60
```

Comment:
can we add these to the autoscaler options? Could see us wanting to change these based on the type of Processor pattern

Reply:
could also just include "pipeline" in the name or something

Reply:
Done. I decided to expose them in flow options. Since we haven't perfectly tuned these numbers it might be nice to allow users to set them especially if we're helping debug any issues with scaling.